provision_client.py

```diff
@@ -0,0 +1,53 @@
+import ujson
```

Comment:
It will be better to import only used functions, not the whole library, to avoid additional memory usage.